src/screens/HomePage/HomePage.tsx

```diff
@@ -72,6 +73,12 @@ export default function HomePage() {
   const exploreSectionRef = useRef(null);
 
   const [menuOpen, setMenuOpen] = useState(false);
+  const [showCompareModal, setShowCompareModal] = useState(false);
+
+  const vaccinationsTableButtonOnClick = () => {
+    window.location.href = '#compare';
```

Comment:
I was gonna' ask why we want to scroll them down to compare, but then I realized it's so that when they close the modal it's not jarring that they get scrolled down to compare...  but *then* I realized that it's so that the vaccinations column is first!

Maybe add a comment to explain? 😁 

Alternatively we could turn `compareShowVaccinationsFirst` into a state variable.. Something like:

```
  const [compareShowVaccinationsFirst, setCompareShowVaccinationsFirst] = useState<boolean>(false);
  useEffect(() => {
    if (location.hash === '#compare') {
      setCompareShowVaccinationsFirst(true);
    }
  }, [location.hash]);
  const vaccinationsTableButtonOnClick = () => {
    setCompareShowVaccinationsFirst(true);
    setShowCompareModal(true);
  };
```

But I'm also down with the hash.

Reply:
switched to a state variable